pages/clusters/overview.md

```diff
@@ -30,8 +50,38 @@ _Clusters_ will now appear in the global navigation.
 
```

Comment:
Keep this section after the clusters GA release, and reword to indicate that this is only relevant (i.e. unclustered whatchimees) for people who created their Buildkite orgs prior to clusters GA.